index.js

```diff
@@ -1,18 +1,24 @@
-'use strict'
+'use strict';
 const fs = require('fs-promise');
+const dateFormat = require('dateformat');
+
 
 module.exports = {
-    handle: function() {
-        return fs.appendFile('/home/rover/hooksreceived.log', 'got file');
+    handle: function(body) {
+        const zen = body.zen;
+        const timestamp = dateFormat(new Date(), 'mmm dd yyyy HH:MM:ss');
```

Comment:
Let's take this to slack and I'll adjust the format in my next PR if that's okay? :)